src/app/lib/app-switcher/app-switcher.component.html

```diff
@@ -5,7 +5,7 @@
      [className]="brandBg"
      (popperOnShown)="buttonOn()"
      (popperOnHidden)="buttonOff()">
-    <img src="./assets/TriFlame.svg">
+    <img height="37px" width="auto" src="./assets/TriFlame.svg">
```

Comment:
There is already sizes defined in the sass file, I would prefer you change them here instead of inline styles that overwrite them.

![image](https://user-images.githubusercontent.com/1934453/35819534-3bd1bcbe-0a60-11e8-879e-3cf38c02724d.png)